.github/workflows/build-release.yml

```diff
@@ -136,8 +168,8 @@ jobs:
           files: |
             exec/android/bundle/release/app-release.aab
             exec/android/flutter-apk/*
-            SyncVault-${{ github.ref_name }}-windows.zip
-            SyncVault-${{ github.ref_name }}-linux.tar.gz
+            ExCode-win.zip
```

Comment:
Revert this and the next line to 
`SyncVault-${{ github.ref_name }}-windows.zip`
and
`SyncVault-${{ github.ref_name }}-linux.tar.gz`

Also add
`SyncVault-${{ github.ref_name }}-macos.tar.gz`